src/RichText/RichText.tsx

```diff
@@ -135,7 +135,7 @@ export const RichText = ({ editor, ...props }: RichTextProps) => {
             : ''
         }${
           editor.initialContent
-            ? `window.initialContent = '${editor.initialContent}';`
+            ? `window.initialContent = ${JSON.stringify(editor.initialContent)};`
```

Comment:
We need to update `initialContent`'s type to accept a string or JSON

<img width="870" alt="Screenshot 2024-06-06 at 15 54 00" src="https://github.com/10play/10tap-editor/assets/36531255/5b4de325-ba50-4093-b6a0-a8ea38f6b9a9">

Reply:
ok I have updated my commit, I added in `string | object`.